ingestion/ingestion_apis/policy_api.py

```diff
@@ -0,0 +1,181 @@
+import logging
+import os
+import json
+import requests
+import glob
+from http import HTTPStatus
+
+from ingestion.ingestion_server.ingestion_globals import IngestionGlobals
+from ingestion.ingestion_apis import config
+from magen_logger.logger_config import LogDefaults
+from magen_rest_apis.server_urls import ServerUrls
+from magen_rest_apis.rest_client_apis import RestClientApis
+from magen_rest_apis.rest_return_api import RestReturn
+
+logger = logging.getLogger(LogDefaults.default_log_name)
+
+
+def create_policy_file(asset, policy_file):
+    """
+    Creates a policy file for the asset. It will actually write the file to disk
+    :param asset: Policy package and Base document name
+    :type asset: String
+    :param policy_file: Destination policy file
+    :type policy_file: String
+    :return: True or False
+    """
+    try:
+        with open(policy_file, 'w+') as file:
+            file.write('package opa.' + asset + '\n\n')
+            file.write('import input as http_api' + '\n')
+            file.write('import data.' + asset + '\n\n')
+            file.write('default allow = false' + '\n\n')
+            file.write('allow {' + '\n\t')
+            file.write('http_api.owner = ' + asset + '[owner]\n\t')
+            file.write('http_api.asset = ' + asset + '[asset_id]\n\t')
+            file.write('http_api.user = ' + asset + '[users][_]\n\t')
+            file.write('http_api.path = ' + asset + '[url]\n')
+            file.write('}')
+        return True
+    except FileExistsError as e:
+        message = 'Failed to create policy {}'.format(policy_file)
+        logger.error(message + str(e))
+        return False
+
+
+def create_base_document(asset_id, owner):
+    """
+    :param asset_id: id of the asset
+    :type asset_id: String
+    :param owner: owner of the asset
+    :type owner: String
+    :return: True/False with json/error message
+    """
+    try:
+        server_urls_instance = ServerUrls().get_instance()
```

Comment:
it works with server_urls now (as opa_client is a part of ingestion). When we move it to a package - server_urls will be an empty instance with no servers addresses, ports set.

It's ok for now, I have an idea how to work around it in a package - we can discuss it later